### PR TITLE
content updates, listed

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

Module: Fundamentals
Week(s): all

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

## Description

Adding more stuff into Fundamentals - will also start adding in examples of mentor notes in tabs (PD don't want to assume mentors are present, which is wise/true)

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
this failed because I changed the build command to test the pagefind search
have now rm 
&& npx pagefind --source "public"

**Comment by SallyMcGrath** [comment]
I'll set to draft until someone looks at the search feature

**Review by illicitonion — APPROVED** [review]

File: content/blocks/backlog/index.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ vocabulary="Backlog"
...
 +++
 
-You will find your backlog in the [Backlog](../backlog/). This is a list of all the tickets for the module. You can also go week by week by looking in the sprint backlog.
+In software development, we break down complex projects into smaller, manageable parts, which we work on for a week or two. These periods are called "sprints."
+
+A sprint backlog is like a to-do list. It lists what the team has decided to work on this sprint. It's chosen from a larger list, usually called the "product backlog," which holds the entire project to-do list.
+
+For your course, the mandatory work has been scheduled already. You will add any other tickets you want to work on to your backlog and schedule them according to your own goals and capacity. Use your planning board to do this.
+
+You will find your backlog in the [Backlog](../backlog/).
```
Comment (40thieves) [inline content/blocks/backlog/index.md:19]: This link appears to be dead for fundamentals, as there is no top-level backlog page. Unclear to me if that's just an oversight in fundamentals, or deliberate in which case this link may have to made conditional somehow.

**Review by 40thieves — COMMENTED** [review]
Maybe it's because there's a big queue of PRs that all interact with each other but there's something odd with this branch for me (locally).

It seems that module blocks aren't working, e.g. here's the day plan for sprint 1: 

image

That [should be showing](https://github.com/CodeYourFuture/curriculum/blob/db0a63e56268917813a47bf0f54a44af291f0d79/content/fundamentals/sprints/1/day-plan/index.md?plain=1#L20-L25) `code-dot-org` and `cyf-blocks-requirements` blocks.

Barring these general notes, the specific changes here LGTM

**Pull request merged**